app/src/main/java/com/eveningoutpost/dexdrip/stats/StatsResult.java

```diff
@@ -110,4 +128,14 @@ public String getAverageUnitised(){
         return "Avg:" + (new DecimalFormat("#.0")).format(avg*Constants.MGDL_TO_MMOLL);
     }
 
+    public String getCapturePercentage(boolean extended){
+        String result =  "Cap:" + ((possibleCaptures>0)?Math.round(getTotalReadings()*100d/possibleCaptures) + "%":"-%");
```

Comment:
Where I added Cap: for the extra status line I don't think it is possible for it to ever get to 100% the use of Math.round in your version here probably will work better. Can we get both in sync in due course?